pkgs/common-updater/generic-updater.nix

```diff
@@ -62,7 +62,7 @@ let
       return 1
     }
 
-    tags=$($version_lister --pname=$pname --attr-path=$attr_path --file="${fileForGitCommands}") || exit 1
+    tags=$(sh -c "$version_lister --pname=$pname --attr-path=$attr_path --file=\"${fileForGitCommands}\"") || exit 1
```

Comment:
Why do we need that extra shell here?

Reply:
The previous version was incorrect for running shell commands:

```ShellSession
$ cmd="echo 'foo   bar'"

$ echo "$(sh -c "$cmd")"
foo   bar

$ echo "$($cmd)"
'foo bar'
```

Maybe there is a nicer way to run commands?